test/data/manifests/rhel_84-x86_64-vhd-boot.json

```diff
@@ -9761,6 +9774,16 @@
       "unbound-anchor.timer",
       "waagent.service"
     ],
+    "sysconfig": {
+      "kernel": {
+        "DEFAULTKERNEL": "kernel",
+        "UPDATEDEFAULT": "yes"
+      },
+      "network": {
+        "NETWORKING": "yes",
+        "NOZEROCONF": "yes"
+      }
+    },
     "timezone": "New_York"
   }
-}
+}
```

Comment:
Lets hope David doesn't see this :P

Reply:
hahaha shhhh. It's the test generator's fault...